iothub_client/samples/iothub_client_sample_upload_to_blob_mb/iothub_client_sample_upload_to_blob_mb_stress.c

```diff
@@ -54,6 +58,7 @@ static IOTHUB_CLIENT_FILE_UPLOAD_GET_DATA_RESULT getDataCallback(IOTHUB_CLIENT_F
                 *data = (const unsigned char*)data_to_upload;
                 *size = block_size;
                 blocks_sent++;
+                (void)printf("."); fflush(stdout);
```

Comment:
Too much decoration. The sample is supposed to be as lean as possible, unless the directive has changed.

Reply:
I don't see how the sample clarity is affected. Actually, not having progress is a huge problem here as uploading 2000 chunks can take minutes and the sample appears as hanging.